ion/src/emscripten/main.cpp

```diff
@@ -2,6 +2,9 @@
 #include "display.h"
 #include "events_keyboard.h"
 #include "../../../apps/global_preferences.h"
+extern "C" {
```

Comment:
Can't we re-use the STL code from blackbox?

Reply:
No real reason to use SDL here. It was just included there